modules/ROOT/pages/inline-css.adoc

```diff
@@ -37,75 +37,154 @@ tinymce.init({
-* A `<style>` in the `<body>` of the page
-** As noted in the `Classic iframe` section, this should not be done, but some browsers will still allow it.
-* A `style` attribute on an {productname} element within the content i.e. inline CSS
+When a {productname} instance is in a shadow root, the {pluginame} should still support
 
-=== Shadow DOM is supported but **not** officially, with some notable limitations
+* CSS specified by the `xref:add-css-options.adoc#content_css[content_css]` option; and
 
-Shadow DOM is **not** officially supported by {productname}, but the feature will work when an editor running in `inline` or `iframe` is contained within a shadow root. 
+* CSS specified by the `xref:add-css-options.adoc#content_style[content_style]` option.
 
-When an `iframe` mode editor is contained with a shadow root, the editor content can be styled the same way, where the expected behavior for the `feature` is that it should not have to consider if the editor is in a shadow root. When an `inline` editor is contained within a shadow root, the stylesheet (`<link>` tags) and `<style>` tags are added directly as children of the shadow root and do not need to be contained within a `<head>` tag. 
 
-When in `inline mode`, the feature checks if the editor is inside a shadow root to ensure it does not try to get `<styles>` that are outside the shadow root.
+==== Classic mode
 
-When an editor is in a shadow root, the feature should still support getting the stylesheets specified via the `content_css` option as well as the CSS styles specified via  the`content_style` option.
+When a {productname} editor in classic mode is contained within a shadow root, CSS applied to the editor’s content will work as expected if the editor can also safely assume it is not operating in a shadow root.
 
-// * some pseudo-class selectors. See the specification for more details
-// (https://notion.so/tinycloud/Technical-specification-7c16d4522d1241ba9e7364880eab3e51#94b1d13ff444492bbfce789f7876f39c)
-// 
-// 2. Vogue only supports `@import` @-rules at this stage.
-// 
-// See the Jira ticket: https://ephocks.atlassian.net/browse/TINY-9265
-// 
-// see the tech spec for @-rule support:
-// https://notion.so/tinycloud/Technical-specification-7c16d4522d1241ba9e7364880eab3e51#555fede3a4ff4a26ad5efdf72bc1af6f.
 
-// This plugin allows extracting the content of the editor with CSS inlined.
-// Support classic and inline modes. Can be used in shadow dom. 
-// Should be able to extract CSS from the stylesheets defined via
-// `content_css` and `font_css` options, and CSS styles defined via `style_css` option.
+==== Inline mode
 
+When a {productname} editor in inline mode is contained within a shadow root, tags pointing to CSS declarations — either `<link>` tags pointing to stylesheet files or `<style>` tags containing specific CSS examples — are added as children of the shadow root. These tags do not need to be contained with the `<head>` tag.
```

Comment:
I don't think this is quite accurate. TinyMCE will only automatically add styles specified via the `content_css` and `content_style` options into the shadow root. All other styles would have to be manually done by the integrator.

Reply:
The two sub-sections: Classic mode and Inline mode have been removed, replaced by a sub-clause in the re-written Shadow DOM support section.